www/index.ejs

```diff
@@ -73,6 +73,7 @@
 
       animation: show 1.5s forwards;
       pointer-events: none;
+      transition: height 0s 0.2s;
```

Comment:
lolz

Reply:
haha yeah, IE11 cannot handle pointer-events: none; (only on IE11, Win7) although caniuse.com says it can :(